### DISCUSSION TITLE
Welcome to i3-volume Discussions!

### DISCUSSION THREAD

**Discussion by hastinbe** [hastinbe/i3-volume#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n1trux** [comment]
Github has weird features :smile:
I guess it makes sense, but it's the first time I see this.

After looking for an hour for a well-written, functional application for doing something simple like setting a volume and trying to fix my own shell script (which stopped working 2 years ago), trying [several CLI-applications](https://wiki.archlinux.org/index.php/PulseAudio#Console) including `pactl`/`pacmd` (didn't support max volume), `alsactl` and [pulseaudio-ctl](https://github.com/graysky2/pulseaudio-ctl) (it wanted to be installed as root and wasn't packaged for Fedora) I finally settled on i3-volume. I'm pretty happy with this neat script, well done!
